modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -348,15 +348,6 @@ else if (cmd.equals(Constants.PREVIEW)) {
 
 			String portletResource = ParamUtil.getString(
 				actionRequest, "portletResource");
-
-			if (Validator.isNotNull(portletResource) ||
-				cmd.equals(Constants.ADD_DYNAMIC)) {
-
-				hideDefaultSuccessMessage(actionRequest);
-
-				MultiSessionMessages.add(
```

Comment:
If you delete all this code the sucess message will appears 2 times :


https://user-images.githubusercontent.com/47524751/223499524-40293f19-db4e-4954-b619-aad25fa7bbb7.mp4



I think that removing only this 
`MultiSessionMessages.add(...`
The issue will be solved

try both scenarios

Thanks